core/trino-spi/src/test/java/io/trino/spi/predicate/TestEquatableValueSet.java

```diff
@@ -121,7 +121,12 @@ public void testMultipleValues()
         assertTrue(equatables.containsValue(2L));
         assertTrue(equatables.containsValue(3L));
         assertFalse(equatables.containsValue(4L));
-        assertEquals(equatables.toString(), "[ 1, 2, 3 ]");
+        assertEquals(
+                equatables.toString(),
+                "EquatableValueSet[type=id, values=3, {1, 2, 3}]");
+        assertEquals(
+                equatables.toString(ToStringSession.INSTANCE, 2),
```

Comment:
use some test SESSION here instead

`ToStringSession` should be used inside `toString()` impl only